Source/WebCore/dom/PointerEvent.h

```diff
@@ -130,6 +130,13 @@ class PointerEvent final : public MouseEvent {
         return (type == eventNames.pointerupEvent || type == eventNames.pointeroutEvent || type == eventNames.pointerleaveEvent || type == eventNames.pointercancelEvent) ? 0 : 1;
     }
 #endif
+    static float pressureForPressureInsensitiveInputDevices(unsigned short buttons)
+    {
+        // https://www.w3.org/TR/pointerevents/#dfn-active-buttons-state
+        bool isInActiveButtonsState = buttons;
+        // https://www.w3.org/TR/pointerevents/#dom-pointerevent-pressure
```

Comment:
This bit doesn't seem implemented: `Note: all pointerup events will have pressure 0.`

Not sure if there's a test for this somewhere.

Reply:
The `pointerevents/pointerevent_attributes_hoverable_pointers.html?mouse` WPT makes an assertion about this too, so I think we have enough test coverage.

```js
if (event.buttons === 0) {
    assert_equals(event.pressure, 0, "pressure is 0 with no buttons pressed");
```